opwen_email_server/actions.py

```diff
@@ -221,7 +221,7 @@ def __init__(self, auth: Union[AzureAuth, NoAuth], raw_email_storage: AzureTextS
         self._raw_email_storage = raw_email_storage
         self._next_task = next_task
 
-    def _action(self, client_id, email, **sendgrid_args):  # type: ignore
+    def _action(self, client_id=None, email=None, **sendgrid_args):  # type: ignore
```

Comment:
I understand that the `client_id` can be None in the service email case. But can the email really ever be None?

If `email` has to be set to None so that connexion can pass the correct parameters, I would at least add a check that returns an invalid response error (HTTP 400) if `email` is None.

Reply:
I made email=None to not switch the order of the parameters and make python happy. I'll add that check.